### PR TITLE
Create create_rule_ui.gs

### PR DESCRIPTION
**Pull request by siddhimahawar**

### CODE REVIEW

File: create_rule_ui.gs [hunk 1]
```diff
@@ -0,0 +1,175 @@
...
+      input.ruleType,
+      input.timeFilter,
+      input.projectFilter,
+      projectId,
+      input.maxLimit,
+      Session.getActiveUser().getEmail()
+    ];
+    active.appendRow(rowContent);
+  } else {
+    var newSheet = SpreadsheetApp.create("Admin Rules For Apps Scripts");
```
Comment (shikhagupta1212) [inline create_rule_ui.gs]: use constant

File: create_rule_ui.gs [hunk 2]
```diff
@@ -0,0 +1,175 @@
...
+    var active = newSheet.getActiveSheet();
+    active.setName("Admin Rules");
```
Comment (shikhagupta1212) [inline create_rule_ui.gs:128]: Same as above

**Review by shikhagupta1212 — APPROVED** [review]

File: create_rule_ui.gs (hunk 1 above)
Comment (siddhimahawar) [reply create_rule_ui.gs]: Updated

File: create_rule_ui.gs (hunk 2 above)
Comment (siddhimahawar) [reply create_rule_ui.gs:128]: updated

**Pull request merged**